.travis.yml

```diff
@@ -1,31 +1,10 @@
----
-sudo: false
-
 os: linux
-
+dist: xenial
+sudo: required
 language: go
-
 go:
   - 1.12.x
-
-before_install:
-  - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-
 install:
-  - dep ensure -v
-
-before_script:
-  - go version
-  - export GOBIN="$GOPATH/bin"
-  - go get golang.org/x/tools/cmd/goimports
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0 && golangci-lint --version
```

Comment:
golangci-lint is an industry standard go linter aggregator and offers far more configurability and reliability than golint.  In addition to basic linting, it's configured to perform a number of other useful checks.  The full list of enabled and disabled linters can be seen by `./hack/go.sh linters`.  Executing the linters can be done with `./hack/go.sh lint`.

Reply:
The config file can be found at [.golangci.yaml](https://github.com/kube-object-storage/lib-bucket-provisioner/blob/master/.golangci.yml)